test/pytorch_test_base.py

```diff
@@ -278,6 +278,7 @@
         'test_upsamplingBicubic2d_correctness_xla',  # FIXME! Got dtypes torch.float32 and torch.float64
         'test_CTCLoss_no_batch_dim_xla',  # Value out of range
         'test_upsamplingBilinear2d_xla',  # precision on GPU/TPU, slow compilation on CPU
+        'test_upsamplingBiMode2d_xla',  # slow compilation on CPU/GPU
```

Comment:
can you explain to me why running the test would require the "compilation"?

Reply:
This refers to the hlo graph compilation. The newly updated test's hlo graph compilation was too slow, making the CI timeout.